update storage task execution tip (#558)
astronomer/docs: astro/resource-reference-aws.md

:::tip

> With the exception of `m5d` nodes, all suppported node types have a maximum of 20GB of storage per node for system use only. If you need locally attached storage for task execution, we recommend modifying your cluster to run `m5d` nodes, which Astronomer provisions with NVMe SSD volumes out-of-the-box.

Reviewer: ```suggestion
With the exception of `m5d` nodes, all suppported node types have a maximum of 20GB of storage per node for system use only. If you need locally attached storage for task execution, we recommend modifying your cluster to run `m5d` nodes, which Astronomer provisions with NVMe SSD volumes out of the box.
```

Reviewer 2: @chomipi88 Why not make the deciding factor here whether the user needs more than 20 GB of storage, and not whether they want locally attached storage or not? I feel like the # of GBs is usually the tipping point? I think I'd say:

```suggestion
With the exception of `m5d` nodes, all suppported node types have a maximum of 20GB of storage per node for system use only. If your tasks require more than 20GB of storage and you want to enable locally attached storage, we recommend modifying your cluster to run `m5d` nodes, which Astronomer provisions with NVMe SSD volumes out-of-the-box.
```

Author: @paolaperaza In this case the storage requirement doesn't matter because we don't want them using the system only disk that is attached. Once we support additional ephemeral storage, then that configuration will be driven by the customer.
